lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
@hudochenkov: Sorry ... try putting `jest.mock("os", () => ({ EOL: "\n" }))` in the specific test file running code where that should be mocked, not in `jest-setup`.